index.html

```diff
@@ -113,6 +113,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *Jen card START ________  -->
+        <div class="card">
+          <p class="name">Jen</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">brite107</a>
```

Comment:
Oh no! You probably want to update this link to go to your Github profile rather than the Github home page. :)